nautobot_ssot/jobs/examples.py

```diff
@@ -318,6 +405,22 @@ def load(self):
             self.add(site)
             self.job.log_debug(message=f"Loaded {site} from remote Nautobot instance")
 
+        prefix_data = requests.get(f"{self.url}/api/ipam/prefixes/", headers=self.headers, params={"limit": 0}).json()
+        prefixes = prefix_data["results"]
+        while prefix_data["next"]:
+            prefix_data = requests.get(prefix_data["next"], headers=self.headers, params={"limit": 0}).json()
+            prefixes.extend(prefix_data["results"])
+        for prefix_entry in prefixes:
+            prefix = self.prefix(
+                prefix=prefix_entry["prefix"],
+                description=prefix_entry["description"],
+                status_slug=prefix_entry["status"]["value"] if prefix_entry["status"] else "active",
```

Comment:
Is there ever a case where `prefix_entry["status"]` isn't present? That would seem like a bug to me.

Reply:
don't think so, I guess it's present always because it's a mandatory field, but I followed the same pattern we had before for site and region. Should I update the others too?

Reply:
Hm, yeah, I guess status shouldn't ever be None for Sites either. Region `parent` and Site `region` can be None, so it makes sense to keep the current logic for them.